packages/cluster/scripts/start-client.sh

```diff
@@ -153,3 +153,11 @@ echo $overcommitment_hugepages >/proc/sys/vm/nr_overcommit_hugepages
 # These variables are passed in via Terraform template interpolation
 /opt/consul/bin/run-consul.sh --client --consul-token "${CONSUL_TOKEN}" --cluster-tag-name "${CLUSTER_TAG_NAME}" --enable-gossip-encryption --gossip-encryption-key "${CONSUL_GOSSIP_ENCRYPTION_KEY}" &
 /opt/nomad/bin/run-nomad.sh --client --consul-token "${CONSUL_TOKEN}" &
+
+# Add alias for ssh-ing to sbx
+echo '_sbx_ssh() {
+  local address=$(dig @localhost $1. A +short 2>/dev/null)
```

Comment:
on my host at least, localhost doesn't return the IP of a running instance but this does:
```
jonas@jtdevorch-client-fkl5:~/infra/packages/orchestrator$ dig @127.0.0.4 test-instance-1 A +short
192.168.2.197

jonas@jtdevorch-client-fkl5:~/infra/packages/orchestrator$ dig @localhost test-instance-1 A +short
<nada>
```

Reply:
looks like there must be something different, works for me on a clean deploy

Reply:
maybe worth noting: im looking for instances created via `make test`